sys-apps/irqbalance/files/irqbalance.service

```diff
@@ -2,10 +2,8 @@
 Description=CPU Interrupt Balancer
 
 [Service]
-ExecStart=/usr/sbin/irqbalance --debug
-# --debug implies a lot of cruft
-# but it's the only way to get foregrounding
-StandardOutput=null
+EnvironmentFile=-/etc/irqbalance.env
+ExecStart=/usr/sbin/irqbalance --foreground $IRQBALANCE_ARGS
```

Comment:
How would I determine that, other than that `systemctl status` shows that irqbalance is running and that there are no related upstream bugs?  `strace` shows some periodic activity (`nanosleep()` followed by reading stuff from `/proc/irq/`).

Without throughly reading through each line in great detail, the source code doesn't look very surprising: both `--debug` and `--foreground` set `foreground_mode=1`, and that is what the rest of the code relies on when deciding whether to daemonize or not.

But really, if you *know* there's an issue with `--foreground`, I would expect to find an upstream bug report from you ;-) (although admittedly, I don't know how long the project has been on GitHub).